requirements.txt

```diff
@@ -1,9 +1,8 @@
 gym >= 0.17.2
-paretoset >= 1.1.2
 numpy >= 1.18.5
 pandas
 requests
-pytorch
```

Comment:
I noticed this distinction before. It's actually pytorch if you do a conda install. I don't think I ever fixed this or tested in in the pip install with this requirements.txt file. Thanks for the catch.